src/Core/Config/ConfigCacheLoader.php

```diff
@@ -157,22 +145,43 @@ public function loadConfigFile($filename, $addon = false)
 	}
 
 	/**
-	 * Loads addons configuration files
+	 * Tries to load the specified core-configuration and returns the config array.
+	 *
+	 * @param string $name The name of the configuration
+	 *
+	 * @return array The config array (empty if no config found)
+	 *
+	 * @throws \Exception if the configuration file isn't readable
+	 */
+	public function loadCoreConfig($name)
+	{
+		if (file_exists($this->configDir . $name . '.config.php')) {
+			return $this->loadConfigFile($this->configDir . $name . '.config.php');
+		} elseif (file_exists($this->configDir . $name . '.ini.php')) {
+			return $this->loadINIConfigFile($this->configDir . $name . '.ini.php');
+		} else {
+			return [];
+		}
+	}
+
+	/**
+	 * Tries to load the specified addon-configuration and returns the config array.
 	 *
 	 * First loads all activated addons default configuration through the load_config hook, then load the local.config.php
```

Comment:
I don't believe this is current anymore.

Reply:
It's working this way, why not?